.github/workflows/deploy_to_github_pages.yml

```
@@ -1,9 +1,21 @@
+# =========== IMPORTANT!! READ THIS!! ============ #
+# This file can be copied to be used in other websites deployments.
+# When you copy this file, make sure to check and modify the fields
+# that here are marked with `MODIFY IF NEEDED`.
+
+# Other related files/folders that you will also need to copy:
+#    - .github/workflows/templates/
+#    - .github/workflows/delete_pr_preview.yml
+# ============================================== #
+
 name: Deploy to GitHub Pages
 on:
   push: #Action fires anytime there is a push to the following branches
     branches:
       - main
-      - development
+      - development # MODIFY IF NEEDED - If you don't have a development branch, you can remove this line.
+    paths:
+      - './**' # MODIFY IF NEEDED - This should be changed to be DOC_DIR/** based on the env variable DOC_DIR below
   pull_request: #Action also fires anytime a PR is (re)opened, closed or synchronized
```

```suggestion
  pull_request: # Action also fires anytime a PR is (re)opened, closed or synchronized
```
Spacing consistency.